src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -6,6 +6,7 @@
 import java.util.Collections;
 import java.util.Iterator;
 import java.util.List;
+import java.util.*;
```

Comment:
Please refer to the [coding standards](https://oss-generic.github.io/process/codingStandards/CodingStandard-Java.html#package-and-import-statements), "imported classes should always be listed explicitly" :)